### DISCUSSION TITLE
Can't manipulate data from multiple panel

### DISCUSSION THREAD

**Discussion by avikkumar1983** [grafana/grafana#46683] [Dashboards, Panels & Visualizations]
I have CPU utilization in 1 panel (Panel 1) & Memory utilization in another panel(Panel 2). I am using AWS athena for datasource. Now I want to use both CPU & Memory data in another panel (Panel 3) & try to do calculations based on that. 

I have tried "Dashboard" as data source in Panel 3. But this only allows me 1 query in panel 3 & let me manipulate that data alone. There is no way I can add another query/datasource to bring my panel 2 data. Please see below that there is no option to add 2nd query.

![image](https://user-images.githubusercontent.com/61217148/158507302-69fbee65-9afa-4539-98f3-29358b916f06.png)

If I use 'Mixed' datasource, it allows me to add multiple query but it doesn't allow me 'Dashboard' as a datasource for individual query. Please see below that there is no dashboard option.
![image](https://user-images.githubusercontent.com/61217148/158507140-79870863-5731-4ada-8734-2b96258efe16.png)

**Comment by ivanahuckova** [comment]
This seems like a feature request that should be converted to discussion, but I am passing to @grafana/user-essentials to evaluate and convert to discussion.

**Comment by joshhunt** [comment]
At the moment you cannot add additional data sources to a panel that uses the "-- Dashboard --" data source to retrieve data from another panel. The main intention behind the "Use results from" panel is to visualise the same data in with multiple panels, rather than to continue to augment the data.

You can apply transformations to the data in the second panel, if they have the functionality you're looking for.

Because this is a feature request, I'll move this over discussions so it can gather the right attention there.

**Reply by avikkumar1983** [reply]
If there are multiple panel data we can build a scoring system applying weightage ( using transformation obviously)in 3rd panel based on multiple seperate metrics. That's why I need this multiple panel data in 1 panel without querying it.
for example- 1 new metric based on cpu and memory panel data
 final score=(cpu+ memory)/2
I dontvwant to mix cpu, memory, final score in 1 panel , rather 3 separate panels.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
